go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/containerd/containerd v1.5.4 // indirect
 	github.com/containers/libtrust v0.0.0-20190913040956-14b96171aa3b
 	github.com/containers/ocicrypt v1.1.2
-	github.com/containers/storage v1.37.0
+	github.com/containers/storage v1.36.3
```

Comment:
This is not acceptable.

(Alternative approaches are being discussed, just noting this here so that it doesn’t get merged as is.)

Reply:
@TomSweeneyRedHat @mheon FYI this is sort of a trouble where we are stuck and discussing an alternative approach.

Reply:
@flouthoc looks like a good Cabal topic.  Could you add it to the list please?

Reply:
@TomSweeneyRedHat I created this PR for `v3.4.2-rhel` but afaik the backport for `v3.4.2-rhel` is not decided so we might not need this PR anymore.